ciao-deploy/README.md

```diff
@@ -0,0 +1,150 @@
+# Deploying a ciao cluster
+
+## Prerequisites
+
+Prior to deploying ciao the following prerequisites must be met:
+
+1. Ensure that there is a user account on the system accessed by key
+authentication with **passwordless** sudo access.
+
+2. If your systems access the internet via proxies that these are correctly
+configured. Docker in particular needs special configuration for network
+proxies.
+
+3. A running ceph cluster along with a ceph keyring that can access rbd
+installed on all machines in the cluster. The [Ceph
+Deployment](http://docs.ceph.com/docs/master/rados/deployment/) tool can be
+used to both build the cluster and deploy the keyring to all nodes.
+
+4. Keystone configured with a service and admin user. An example of how to do
+this using a docker container is shown by
+[singlevm](https://github.com/01org/ciao/blob/master/testutil/singlevm/setup_keystone.sh).
+Make note of the password for the admin and service (csr) users as you'll need
+those for the deployment tool.
+
+## Setting up the master node of the cluster
+
+All deployment is driven from the master node of the cluster. On this machine
+you will need to install Go 1.8. Instructions for doing this can be found [on
+the Go website](https://www.golang.org)
+
+Next download, install and build ciao:
+
+```
+go get -u github.com/01org/ciao/./...
+```
+
+Please ensure you have `$GOPATH/bin` in your `$PATH`. If you have not specified
+a `$GOPATH` this will default to `$HOME/go`
+
+Now setup the master node. The parameters below are the minimum that you will
```

Comment:
Now setup the master node using ciao-deploy.

Otherwise it's not really clear what commands these options are for.